src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -108,15 +99,9 @@ public function getQueryBuilderPartsForCachingHash($queryBuilder)
         return false;
     }
 
-    public function __construct(ManagerRegistry $registry, PropertyAccessorInterface $propertyAccessor = null, ChoiceListFactoryInterface $choiceListFactory = null)
+    public function __construct(ManagerRegistry $registry)
```

Comment:
Given a class that extends of `DoctrineType` (currently) if the `__constructor()` was overwritten would look like this:
```php
public function __construct(ManagerRegistry $registry, PropertyAccessorInterface $propertyAccessor = null, ChoiceListFactoryInterface $choiceListFactory = null, $extraData) 
{
    parent::__constructor($registry, $propertyAccessor, $choiceListFactory);

    $this->extraData = $extraData;
}
```
It doesn't break after this PR.